Les sous-répertoires mISDNorig/autom4te.cache et mISDN/autom4te.cache sont identiques
diff -u mISDNorig/config.log mISDN/config.log
--- mISDNorig/config.log	2019-11-05 14:13:31.076443618 +0100
+++ mISDN/config.log	2019-11-04 22:49:16.333443427 +0100
@@ -4,7 +4,7 @@
 It was created by mISDNkernel configure 2.0.36, which was
 generated by GNU Autoconf 2.69.  Invocation command line was
 
-  $ ./configure --no-create --no-recursion
+  $ ./configure 
 
 ## --------- ##
 ## Platform. ##
@@ -49,6 +49,7 @@
 configure:2153: checking for a thread-safe mkdir -p
 configure:2192: result: /usr/bin/mkdir -p
 configure:2199: checking for gawk
+configure:2215: found /usr/bin/gawk
 configure:2226: result: gawk
 configure:2237: checking whether make sets $(MAKE)
 configure:2259: result: yes
@@ -59,8 +60,8 @@
 configure:2508: result: gcc
 configure:2737: checking for C compiler version
 configure:2746: gcc --version >&5
-gcc (Debian 4.8.4-1) 4.8.4
-Copyright (C) 2013 Free Software Foundation, Inc.
+gcc (Debian 9.2.1-16) 9.2.1 20191030
+Copyright (C) 2019 Free Software Foundation, Inc.
 This is free software; see the source for copying conditions.  There is NO
 warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 
@@ -68,22 +69,24 @@
 configure:2746: gcc -v >&5
 Using built-in specs.
 COLLECT_GCC=gcc
-COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-linux-gnu/4.8/lto-wrapper
+COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-linux-gnu/9/lto-wrapper
+OFFLOAD_TARGET_NAMES=nvptx-none:hsa
+OFFLOAD_TARGET_DEFAULT=1
 Target: x86_64-linux-gnu
-Configured with: ../src/configure -v --with-pkgversion='Debian 4.8.4-1' --with-bugurl=file:///usr/share/doc/gcc-4.8/README.Bugs --enable-languages=c,c++,java,go,d,fortran,objc,obj-c++ --prefix=/usr --program-suffix=-4.8 --enable-shared --enable-linker-build-id --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.8 --libdir=/usr/lib --enable-nls --with-sysroot=/ --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --enable-gnu-unique-object --disable-libmudflap --enable-plugin --with-system-zlib --disable-browser-plugin --enable-java-awt=gtk --enable-gtk-cairo --with-java-home=/usr/lib/jvm/java-1.5.0-gcj-4.8-amd64/jre --enable-java-home --with-jvm-root-dir=/usr/lib/jvm/java-1.5.0-gcj-4.8-amd64 --with-jvm-jar-dir=/usr/lib/jvm-exports/java-1.5.0-gcj-4.8-amd64 --with-arch-directory=amd64 --with-ecj-jar=/usr/share/java/eclipse-ecj.jar --enable-objc-gc --enable-multiarch --with-arch-32=i586 --with-abi=m64 --with-multilib-list=m32,m64,mx32 --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
+Configured with: ../src/configure -v --with-pkgversion='Debian 9.2.1-16' --with-bugurl=file:///usr/share/doc/gcc-9/README.Bugs --enable-languages=c,ada,c++,go,brig,d,fortran,objc,obj-c++,gm2 --prefix=/usr --with-gcc-major-version-only --program-suffix=-9 --program-prefix=x86_64-linux-gnu- --enable-shared --enable-linker-build-id --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --libdir=/usr/lib --enable-nls --enable-bootstrap --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --with-default-libstdcxx-abi=new --enable-gnu-unique-object --disable-vtable-verify --enable-plugin --enable-default-pie --with-system-zlib --with-target-system-zlib=auto --enable-objc-gc=auto --enable-multiarch --disable-werror --with-arch-32=i686 --with-abi=m64 --with-multilib-list=m32,m64,mx32 --enable-multilib --with-tune=generic --enable-offload-targets=nvptx-none,hsa --without-cuda-driver --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
 Thread model: posix
-gcc version 4.8.4 (Debian 4.8.4-1) 
+gcc version 9.2.1 20191030 (Debian 9.2.1-16) 
 configure:2757: $? = 0
 configure:2746: gcc -V >&5
 gcc: error: unrecognized command line option '-V'
 gcc: fatal error: no input files
 compilation terminated.
-configure:2757: $? = 4
+configure:2757: $? = 1
 configure:2746: gcc -qversion >&5
-gcc: error: unrecognized command line option '-qversion'
+gcc: error: unrecognized command line option '-qversion'; did you mean '--version'?
 gcc: fatal error: no input files
 compilation terminated.
-configure:2757: $? = 4
+configure:2757: $? = 1
 configure:2777: checking whether the C compiler works
 configure:2799: gcc    conftest.c  >&5
 configure:2803: $? = 0
@@ -124,8 +127,6 @@
 configure:3237: result: yes
 configure:3257: checking whether make supports the include directive
 configure:3272: make -f confmf.GNU && cat confinc.out
-make[1]: Entering directory '/root/mISDNorig'
-make[1]: Leaving directory '/root/mISDNorig'
 this is the am__doit target
 configure:3275: $? = 0
 configure:3294: result: yes (GNU style)
@@ -140,20 +141,46 @@
 configure:3612: checking for gawk
 configure:3639: result: gawk
 configure:3654: checking for find
+configure:3672: found /usr/bin/find
 configure:3685: result: /usr/bin/find
 configure:3702: checking for diff
+configure:3720: found /usr/bin/diff
 configure:3733: result: /usr/bin/diff
 configure:3750: checking for patch
+configure:3768: found /usr/bin/patch
 configure:3781: result: /usr/bin/patch
 configure:3798: checking for ifnames
+configure:3816: found /usr/bin/ifnames
 configure:3829: result: /usr/bin/ifnames
 configure:3842: checking for sort
+configure:3860: found /usr/bin/sort
 configure:3873: result: /usr/bin/sort
 configure:3957: Configured for 5.3  kernel build /lib/modules/5.3.0-kali1-amd64/build source /lib/modules/5.3.0-kali1-amd64/source
 configure:4071: checking that generated files are newer than configure
 configure:4077: result: done
 configure:4100: creating ./config.status
 
+## ---------------------- ##
+## Running config.status. ##
+## ---------------------- ##
+
+This file was extended by mISDNkernel config.status 2.0.36, which was
+generated by GNU Autoconf 2.69.  Invocation command line was
+
+  CONFIG_FILES    = 
+  CONFIG_HEADERS  = 
+  CONFIG_LINKS    = 
+  CONFIG_COMMANDS = 
+  $ ./config.status 
+
+on kali
+
+config.status:847: creating Makefile
+config.status:847: creating standalone/Makefile
+config.status:847: creating config.h
+config.status:1028: config.h is unchanged
+config.status:1076: executing depfiles commands
+
 ## ---------------- ##
 ## Cache variables. ##
 ## ---------------- ##
@@ -165,20 +192,20 @@
 ac_cv_env_CFLAGS_value=
 ac_cv_env_CPPFLAGS_set=
 ac_cv_env_CPPFLAGS_value=
-ac_cv_env_DIFF_set=set
-ac_cv_env_DIFF_value=/usr/bin/diff
-ac_cv_env_FIND_set=set
-ac_cv_env_FIND_value=/usr/bin/find
-ac_cv_env_IFNAMES_set=set
-ac_cv_env_IFNAMES_value=/usr/bin/ifnames
+ac_cv_env_DIFF_set=
+ac_cv_env_DIFF_value=
+ac_cv_env_FIND_set=
+ac_cv_env_FIND_value=
+ac_cv_env_IFNAMES_set=
+ac_cv_env_IFNAMES_value=
 ac_cv_env_LDFLAGS_set=
 ac_cv_env_LDFLAGS_value=
 ac_cv_env_LIBS_set=
 ac_cv_env_LIBS_value=
-ac_cv_env_PATCH_set=set
-ac_cv_env_PATCH_value=/usr/bin/patch
-ac_cv_env_SORT_set=set
-ac_cv_env_SORT_value=/usr/bin/sort
+ac_cv_env_PATCH_set=
+ac_cv_env_PATCH_value=
+ac_cv_env_SORT_set=
+ac_cv_env_SORT_value=
 ac_cv_env_build_alias_set=
 ac_cv_env_build_alias_value=
 ac_cv_env_host_alias_set=
@@ -288,7 +315,7 @@
 htmldir='${docdir}'
 includedir='${prefix}/include'
 infodir='${datarootdir}/info'
-install_sh='${SHELL} /root/mISDNorig/install-sh'
+install_sh='${SHELL} /root/mISDN/install-sh'
 libdir='${exec_prefix}/lib'
 libexecdir='${exec_prefix}/libexec'
 localedir='${datarootdir}/locale'
@@ -321,42 +348,3 @@
 #define VERSION "2.0.36"
 
 configure: exit 0
-
-## ---------------------- ##
-## Running config.status. ##
-## ---------------------- ##
-
-This file was extended by mISDNkernel config.status 2.0.36, which was
-generated by GNU Autoconf 2.69.  Invocation command line was
-
-  CONFIG_FILES    = 
-  CONFIG_HEADERS  = 
-  CONFIG_LINKS    = 
-  CONFIG_COMMANDS = 
-  $ ./config.status 
-
-on kali
-
-config.status:847: creating Makefile
-config.status:847: creating standalone/Makefile
-config.status:847: creating config.h
-config.status:1028: config.h is unchanged
-config.status:1076: executing depfiles commands
-
-## ---------------------- ##
-## Running config.status. ##
-## ---------------------- ##
-
-This file was extended by mISDNkernel config.status 2.0.36, which was
-generated by GNU Autoconf 2.69.  Invocation command line was
-
-  CONFIG_FILES    = 
-  CONFIG_HEADERS  = 
-  CONFIG_LINKS    = 
-  CONFIG_COMMANDS = 
-  $ ./config.status config.h
-
-on kali
-
-config.status:847: creating config.h
-config.status:1028: config.h is unchanged
diff -u mISDNorig/config.status mISDN/config.status
--- mISDNorig/config.status	2019-11-05 14:13:30.512443602 +0100
+++ mISDN/config.status	2019-11-04 22:49:16.193443423 +0100
@@ -427,7 +427,7 @@
 
 Report bugs to <i4ldeveloper@listserv.isdn4linux.de>."
 
-ac_cs_config="'FIND=/usr/bin/find' 'DIFF=/usr/bin/diff' 'PATCH=/usr/bin/patch' 'IFNAMES=/usr/bin/ifnames' 'SORT=/usr/bin/sort'"
+ac_cs_config=""
 ac_cs_version="\
 mISDNkernel config.status 2.0.36
 configured by ./configure, generated by GNU Autoconf 2.69,
@@ -437,7 +437,7 @@
 This config.status script is free software; the Free Software Foundation
 gives unlimited permission to copy, distribute and modify it."
 
-ac_pwd='/root/mISDNorig'
+ac_pwd='/root/mISDN'
 srcdir='.'
 INSTALL='/usr/bin/install -c'
 MKDIR_P='/usr/bin/mkdir -p'
@@ -519,7 +519,7 @@
 fi
 
 if $ac_cs_recheck; then
-  set X /bin/bash './configure'  'FIND=/usr/bin/find' 'DIFF=/usr/bin/diff' 'PATCH=/usr/bin/patch' 'IFNAMES=/usr/bin/ifnames' 'SORT=/usr/bin/sort' $ac_configure_extra_args --no-create --no-recursion
+  set X /bin/bash './configure'  $ac_configure_extra_args --no-create --no-recursion
   shift
   $as_echo "running CONFIG_SHELL=/bin/bash $*" >&6
   CONFIG_SHELL='/bin/bash'
@@ -659,7 +659,7 @@
 S["MKDIR_P"]="/usr/bin/mkdir -p"
 S["INSTALL_STRIP_PROGRAM"]="$(install_sh) -c -s"
 S["STRIP"]=""
-S["install_sh"]="${SHELL} /root/mISDNorig/install-sh"
+S["install_sh"]="${SHELL} /root/mISDN/install-sh"
 S["MAKEINFO"]="makeinfo"
 S["AUTOHEADER"]="autoheader"
 S["AUTOMAKE"]="automake-1.16"
Les sous-répertoires mISDNorig/drivers et mISDN/drivers sont identiques
Les sous-répertoires mISDNorig/.git et mISDN/.git sont identiques
Les sous-répertoires mISDNorig/include et mISDN/include sont identiques
diff -u mISDNorig/Makefile mISDN/Makefile
--- mISDNorig/Makefile	2019-11-05 14:13:30.632443605 +0100
+++ mISDN/Makefile	2019-11-04 22:49:16.253443424 +0100
@@ -253,10 +253,10 @@
 SORT = /usr/bin/sort
 STRIP = 
 VERSION = 2.0.36
-abs_builddir = /root/mISDNorig
-abs_srcdir = /root/mISDNorig
-abs_top_builddir = /root/mISDNorig
-abs_top_srcdir = /root/mISDNorig
+abs_builddir = /root/mISDN
+abs_srcdir = /root/mISDN
+abs_top_builddir = /root/mISDN
+abs_top_srcdir = /root/mISDN
 ac_ct_CC = gcc
 am__include = include
 am__leading_dot = .
@@ -275,7 +275,7 @@
 htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
-install_sh = ${SHELL} /root/mISDNorig/install-sh
+install_sh = ${SHELL} /root/mISDN/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
Les sous-répertoires mISDNorig/patches et mISDN/patches sont identiques
Les sous-répertoires mISDNorig/scripts et mISDN/scripts sont identiques
Les sous-répertoires mISDNorig/series et mISDN/series sont identiques
Les sous-répertoires mISDNorig/standalone et mISDN/standalone sont identiques
